build/lib/stats.ts

```diff
@@ -72,77 +70,3 @@ export function createStatsStream(group: string, log?: boolean): es.ThroughStrea
 		this.emit('end');
 	});
 }
-
-export function submitAllStats(productJson: any, commit: string): Promise<boolean> {
```

Comment:
Removed in https://github.com/microsoft/vscode/commit/7cde1becc6ff7cf31e3f950c390566a5c36a8479